USERGUIDE.md

```diff
@@ -481,15 +497,34 @@ Host app will use the defined interfaces to retrieve these details from manifest
 Retrieve the meta-data of a MiniApp using the following method,
 
 ```swift
+var theManifest: MiniAppManifest?
+
 MiniApp.shared().getMiniAppManifest(miniAppId: miniAppId, miniAppVersion: miniAppVersionId) { (result) in
     switch result {
         case .success(let manifestData):
             // Retrieve the custom key/value pair like the following.
+            theManifest = manifestData
```

Comment:
Looks like `theManifest` isn't being used for anything in this example?

Reply:
Not really, I remove it